src/UI/Buyer/src/app/components/checkout/checkout/checkout.component.ts

```diff
@@ -334,8 +347,12 @@ export class OCMCheckout implements OnInit {
 
   getPaymentError(errorReason: string): string {
     const reason = errorReason.replace('AVS', 'Address Verification') // AVS isn't likely something to be understood by a layperson
-    const part1 = this.translate.instant('ERRORS.CREDIT_CARD_AUTH.MESSAGE_PART1')
-    const part2 = this.translate.instant('ERRORS.CREDIT_CARD_AUTH.MESSAGE_PART2')
+    const part1 = this.translate.instant(
+      'ERRORS.CREDIT_CARD_AUTH.MESSAGE_PART1'
+    )
+    const part2 = this.translate.instant(
+      'ERRORS.CREDIT_CARD_AUTH.MESSAGE_PART2'
+    )
```

Comment:
I know you didn't add this code  but would you mind adding these values to the translation table? I'm not seeing them [included here](https://github.com/ordercloud-api/headstart/blob/development/src/Middleware/src/Headstart.Common/Assets/english-translations.json)